aiida_workgraph/engine/workgraph.py

```diff
@@ -1066,40 +1119,74 @@ def run_tasks(self, names: t.List[str], continue_workgraph: bool = True) -> None
                 self.set_task_state_info(name, "state", "FINISHED")
                 self.update_parent_task_state(name)
                 self.continue_workgraph()
+            elif task["metadata"]["node_type"].upper() in ["AWAITABLE"]:
+                for key in self.ctx._tasks[name]["metadata"]["args"]:
+                    kwargs.pop(key, None)
+                awaitable_target = asyncio.ensure_future(
+                    self.run_executor(executor, args, kwargs, var_args, var_kwargs),
+                    loop=self.loop,
+                )
+                awaitable = self.construct_awaitable_function(name, awaitable_target)
+                self.set_task_state_info(name, "state", "RUNNING")
```

Comment:
If we report the finished state should we not also log somewhere the running state? 
```suggestion
                self.set_task_state_info(name, "state", "RUNNING")
                self.report(f"Task: {name} is running.", )
```
Maybe on a different log level like debug, but for that I think one needs to write a new function, but the report function is fairly easy to implement. https://github.com/aiidateam/aiida-core/blob/fb3686271fcdeb5506838a5a3069955546b05460/src/aiida/engine/processes/process.py#L589-L600

Reply:
At the top of the `run_tasks` function, it reports that to run the task and also shows the type of the task. Thus I prefer not to report the `running` state.